Packs/GoogleCloudLogging/Integrations/GoogleCloudLogging/GoogleCloudLogging.yml

```diff
@@ -0,0 +1,211 @@
+category: Cloud Services
+commonfields:
+  id: GoogleCloudLogging
+  version: -1
+configuration:
+- displaypassword: Service Account JSON
+  additionalinfo: User's Service Account key in JSON format
+  name: credentials
+  required: true
+  hiddenusername: true
+  type: 9
+  section: Connect
+- display: Trust any certificate (not secure)
+  name: insecure
+  required: false
+  type: 8
+  section: Connect
+  advanced: true
+- display: Use system proxy settings
+  name: proxy
+  required: false
+  type: 8
+  section: Connect
+  advanced: true
+description: With Google Cloud Logging, users can centralize all their logs in a single location, making it easier to troubleshoot issues and gain insights from their data.
+display: Google Cloud Logging
+name: GoogleCloudLogging
+script:
+  commands:
+  - arguments:
+    - default: false
+      description: 'A comma-separated list of names of parent resources from which to retrieve log entries. Acceptable values are: projects/[PROJECT_ID], organizations/[ORGANIZATION_ID], billingAccounts/[BILLING_ACCOUNT_ID], folders/[FOLDER_ID]. A maximum of 100 resources may be specified.'
+      isArray: false
+      name: resource_names
+      required: true
+      secret: false
+    - default: false
+      description: When specified, the results returned limited to log entries that match the filter. An empty filter matches all log entries in the resources listed in resourceNames. Referencing a parent resource that is not listed in resourceNames will cause the filter to return no results. The maximum length of a filter is 20,000 characters.
+      isArray: false
+      name: filter
+      required: false
+      secret: false
+    - default: false
+      defaultValue: timestamp asc
+      description: How the results should be sorted. Presently, the only permitted values are "timestamp asc" (default) and "timestamp desc". The first option returns entries in order of increasing values of LogEntry.timestamp (oldest first), and the second option returns entries in order of decreasing timestamps (newest first). Entries with equal timestamps are returned in order of their insertId values.
```

Comment:
Why it is not a list of default options? please change that.
Also, no need to explain here the order and everything, it is self explanatory. 
Just give the descriptions, the options and what's the default